src/main/java/com/sparrowwallet/sparrow/control/TitledDescriptionPane.java

```diff
@@ -178,4 +197,27 @@ protected static int getAccount(Wallet wallet, KeyDerivation requiredDerivation)
 
         return account;
     }
+
+    // Helper method to create a wrapped Label with a specified maxWidth
+    private Label createWrappedLabel(String text) {
+        Label label = new Label(text);
+        label.setWrapText(true);
+        label.setMaxWidth(400);
+        return label;
+    }
+
+    // Helper method to create a Hyperlink
+    private Hyperlink createHyperlink(String url) {
+        Hyperlink hyperlink = new Hyperlink(url);
+        hyperlink.setMaxWidth(400);  // Set maximum width for wrapping
+        hyperlink.setWrapText(true);  // Ensure text wrapping in the hyperlink
+        hyperlink.setOnAction(e -> {
+            try {
+                java.awt.Desktop.getDesktop().browse(new java.net.URI(url));
+            } catch (Exception ex) {
+               // TODO: handle failure to launch link
+            }
+        });
```

Comment:
Please change to `hyperlink.setOnAction(_ -> AppServices.get().getApplication().getHostServices().showDocument(url));`